src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogAgentClient.java

```diff
@@ -273,42 +275,44 @@ private boolean reinitializeTracer(boolean force) {
         }
 
         try {
-            logger.info("Re/Initialize Datadog-Plugin Tracer: hostname = " + this.hostname + ", traceCollectionPort = " + this.traceCollectionPort);
-            final DDTracer.DDTracerBuilder tracerBuilder = DDTracer.builder();
-            tracerBuilder
-                    .sampler(new ForcePrioritySampler(PrioritySampling.SAMPLER_KEEP))
-                    .writer(DDAgentWriter.builder()
-                    .agentHost(this.hostname)
-                    .traceAgentPort(traceCollectionPort)
-                    .monitoring(Monitoring.DISABLED)
-                    .prioritization(Prioritization.ENSURE_TRACE).build());
-
-            final Tracer ddTracer = tracerBuilder.build();
-            traceBuildLogic = new DatadogTraceBuildLogic(ddTracer);
-            tracePipelineLogic = new DatadogTracePipelineLogic(ddTracer);
+            this.stopAgentHttpClient();
+            logger.info("Re/Initialize Datadog-Plugin Agent Http Client");
+            final URL tracesURL = new URL("http://"+this.hostname+":"+this.traceCollectionPort+"/v0.3/traces");
```

Comment:
this seems important enough to have it's own function or a constant with a template string

Reply:
created